src/Umbraco.Web.UI.Client/src/views/prevalueeditors/boolean.controller.js

```diff
@@ -1,4 +1,7 @@
 ﻿angular.module("umbraco").controller("Umbraco.PrevalueEditors.BooleanController",
     function ($scope) {
+        if ($scope.model.value === 1 || $scope.model.value === "1" || $scope.model.value === true) {
```

Comment:
@Shazwazza has something changed for booleans in V8 that we need to try & get all the variations of true?
Also I assume the UI/view expects a string `1` as opposed to a true JS  bool

Reply:
@warrenbuckley yup so the way that pre-values are persisted is completely different in v8. We used to have an odd pre-value DB table which didn't really make any sense. Now pre-values are just stored as a json blob in the same row as the data type in the data type DB table which makes DB lookups faster too. Because the pre-value model for booleans is strongly typed, it means that the JSON for boolean values is serialized to a real boolean 'true' value whereas before these were not strongly typed and we cheated with boolean pre-values to use strings of "1" and "0" just because it was easier at that time. So now the boolean pre-value editor is updated to support any boolean type value as you can see in this code.